plug-in-development/index.html

```diff
@@ -53,7 +53,7 @@ <h3 class="section" indent="1" id="launch-configuration">Launch configuration</h
 </p>
 <pre class="brush:java">
 fun main(argv: Array&lt;String&gt;) {
-    com.beust.kobalt.main(argv)
+    com.beust.kobalt.internal.main(argv)
```

Comment:
Ok, I sorted that out: that code does work if you depend on the ```kobalt.jar```. But for that to work, I think the documentation on how to depend on that (instead of the API) should be improved.